packages/babel-plugin-transform-react-jsx/src/create-plugin.js

```diff
@@ -30,7 +30,11 @@ export default function createPlugin({ name, development }) {
       // TODO (Babel 8): It should throw if this option is used with the automatic runtime
       filter,
 
-      runtime: RUNTIME_DEFAULT = development ? "automatic" : "classic",
+      runtime: RUNTIME_DEFAULT = process.env.BABEL_8_BREAKING
+        ? "automatic"
+        : development
+        ? "automatic"
+        : "classic",
```

Comment:
This is not 
```js
     RUNTIME_DEFAULT = process.env.BABEL_8_BREAKING || development
        ? "automatic"
        : "classic",
```

So that at build time it is easily converted to `development ? "automatic" : "classic"` or `"automatic"` depending on `BABEL_8_BREAKING`.